code-of-conduct-languages/In.md

```diff
@@ -0,0 +1,22 @@
+# योगदानकर्ता आचार संहिता
+
+इस परियोजना के योगदानकर्ताओं और अनुरक्षकों के रूप में, और एक खुले और स्वागत करने वाले समुदाय को बढ़ावा देने के हित में, हम उन सभी लोगों का सम्मान करने की प्रतिज्ञा करते हैं जो मुद्दों की रिपोर्ट करने, फीचर अनुरोध पोस्ट करने, दस्तावेज अपडेट करने, पुल अनुरोध या पैच सबमिट करने और अन्य गतिविधियों के माध्यम से योगदान करते हैं।
+
+अनुभव, लिंग, लिंग पहचान और अभिव्यक्ति, यौन अभिविन्यास, विकलांगता, व्यक्तिगत उपस्थिति, शरीर का आकार, जाति, जातीयता, आयु, धर्म, या के स्तर की परवाह किए बिना, हम इस परियोजना में भागीदारी को सभी के लिए उत्पीड़न-मुक्त अनुभव बनाने के लिए प्रतिबद्ध हैं। राष्ट्रीयता।
```

Comment:
The word राष्ट्रीयता is misplaced. Sentence should be :
आयु, धर्म, या राष्ट्रीयता के स्तर की परवाह किए बिना